docs/swagger.yaml

```
@@ -1244,7 +1244,64 @@ paths:
         5XX:
           description: Unexpected error
       security: 
+        - bearerAuth: []  
+  /v1/timeline:
+    post:
+      tags:
+        - timeline
```

```suggestion
        - timelines
```
然后文档最前面的 `tags:` 里应该要先声明 `timelines`